src/App/Pages/Accounts/LoginPasswordlessViewModel.cs

```diff
@@ -0,0 +1,191 @@
+                SetProperty(ref _email, value);
+            }
+        }
+
+        public string FingerprintPhrase
+        {
+            get => _fingerprintPhrase;
+            set
+            {
+                FingerprintPhraseFormatted = CreateFingerprintPhrase(value);
+                SetProperty(ref _fingerprintPhrase, value);
+            }
+        }
+
+        public FormattedString FingerprintPhraseFormatted
+        {
+            get => _fingerprintPhraseFormatted;
+            set => SetProperty(ref _fingerprintPhraseFormatted, value);
+        }
+
+        public string LogInAttempByLabel
+        {
+            get => _logInAttempByLabel;
+            set => SetProperty(ref _logInAttempByLabel, value);
+        }
+
+        public string DeviceType
+        {
+            get => _deviceType;
+            set => SetProperty(ref _deviceType, value);
+        }
+
+        public string IpAddress
+        {
+            get => _ipAddress;
+            set => SetProperty(ref _ipAddress, value);
+        }
+
+        public string NearLocation
+        {
+            get => _nearLocation;
+            set => SetProperty(ref _nearLocation, value);
+        }
+
+        public DateTime RequestDate
+        {
+            get => _requestDate;
+            set
+            {
+                TimeOfRequestText = CreateRequestDate();
+                SetProperty(ref _requestDate, value);
+            }
+        }
+
+        public string TimeOfRequestText
+        {
+            get => _timeOfRequest;
+            set
+            {
+                SetProperty(ref _timeOfRequest, value);
+            }
+        }
+
+        private FormattedString CreateFingerprintPhrase(string fingerprintPhrase)
+        {
+            var fingerprintList = fingerprintPhrase.Split('-').ToList();
+            var fs = new FormattedString();
+            var lastFingerprint = fingerprintList.LastOrDefault();
+
+            foreach (var fingerprint in fingerprintList)
+            {
+                fs.Spans.Add(new Span
+                {
+                    Text = fingerprint
+                });
+
+                if (fingerprint == lastFingerprint)
+                {
+                    break;
+                }
+
+                fs.Spans.Add(new Span
+                {
+                    Text = "-",
+                    TextColor = ThemeManager.GetResourceColor("DangerColor")
+                });
+            }
+
+            return fs;
+        }
+
+        private string CreateRequestDate()
+        {
+            var minutesSinceRequest = RequestDate.ToUniversalTime().Minute - DateTime.UtcNow.Minute;
+            if (minutesSinceRequest < 5)
+            {
+                return AppResources.JustNow;
+            }
+            if (minutesSinceRequest < 59)
+            {
+                return $"{minutesSinceRequest} {AppResources.MinutesAgo}";
```

Comment:
Not all languages have the format "x minutes ago", so this should be a string format:
`string.Format(AppResources.XMinutesAgo, minutesSinceRequest)`
and in the AppResources `XMinutesAgo` would be: `{0} minutes ago`